Makefile

```diff
@@ -20,4 +20,4 @@ release: clean
 	pipenv run twine upload dist/*
 
 black:
-	black --target-version py27 .
+	pipenv run black --target-version py34 .
```

Comment:
black is now stable but apparently doesn't support python 2.7 anymore. This also leads to the string changes in `config_tests.py`